src/stp/new_roles/Attacker.cpp

```diff
@@ -0,0 +1,18 @@
+//
+// Created by jordi on 17-03-20.
+//
+
+#include "stp/new_roles/Attacker.h"
+#include "stp/new_tactics/GetBall.h"
+#include "stp/new_tactics/DriveWithBall.h"
+#include "stp/new_tactics/KickAtPos.h"
+
+namespace rtt::ai::stp::role {
+
+Attacker::Attacker(std::string name) : Role(std::move(name)) {
+    // create state machine and initializes the first state
+    robotTactics = collections::state_machine<Tactic, Status, StpInfo>{tactic::GetBall()/*, tactic::DriveWithBall()*/, tactic::KickAtPos()};
+    robotTactics.initialize();
```

Comment:
maybe we should add this to the constructor of state_machine, initialize the first one?

Reply:
I think that would work better, I will make an issue out of it